services-geojson/build.gradle

```diff
@@ -1,12 +1,71 @@
-apply plugin: 'java-library'
+import org.jetbrains.kotlin.gradle.plugin.mpp.apple.XCFrameworkConfig
 
-dependencies {
-  // Gson
-  api libs.gson
+plugins {
+    alias(libs.plugins.kotlin.multiplatform)
+    alias(libs.plugins.kotlinx.serialization)
+    id("maven-publish")
+    alias(libs.plugins.kotlin.cocoapods)
+}
+
+kotlin {
+    jvmToolchain(17)
+
+    jvm()
+
+    def xcf = new XCFrameworkConfig(project)
+    [iosX64(), iosArm64(), iosSimulatorArm64()].forEach { target ->
+        target.binaries.framework { framework ->
+            baseName = "maplibre-geojson"
+            xcf.add(framework)
+        }
+    }
```

Comment:
It would be good to additionally support `js` and `wasmJs` targets (MapLibre Compose currently builds for JS and I intend to eventually build for Wasm)